src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+	Book findByAuthor(String author);
```

Comment:
We prefer to return an Optional<Book> instead only a Book just to handle in an easier way the absence of a Book by this criteria (for instance, if we search for an author that has not published any book at all).